MatrixSDK/Data/Store/MXStore.h

```diff
@@ -324,6 +324,21 @@
 - (void)close;
 
 
+#pragma mark - Media repository
```

Comment:
Note: these follow an `@optional` further up, and are only implemented in `MXFileStore`.